rb-files/app.rb

```diff
@@ -97,17 +84,19 @@ def create_rental
   def list_rentals_by_person_id
     print 'ID of person: '
     id = gets.chomp.to_i
```

Comment:
- [ ] The previous collector refers to this part of the code, All requests for input and input validation in your methods should've been placed in the main.rb file and/or classes/methods that are responsible for requesting or validating input. 
which means that this part of the code should be moved to a class or method which specifically handles user input.